packages/prisma/schema.prisma

```diff
@@ -376,17 +376,28 @@ enum WebhookTriggerEvents {
   BOOKING_CANCELLED
 }
 
+enum SubscriptionType {
+  WEBHOOK
+  ZAPIER
+}
+
 model Webhook {
-  id              String                 @id @unique
-  userId          Int?
-  eventTypeId     Int?
-  subscriberUrl   String
-  payloadTemplate String?
-  createdAt       DateTime               @default(now())
-  active          Boolean                @default(true)
-  eventTriggers   WebhookTriggerEvents[]
-  user            User?                  @relation(fields: [userId], references: [id], onDelete: Cascade)
-  eventType       EventType?             @relation(fields: [eventTypeId], references: [id], onDelete: Cascade)
+  id               String                 @id @unique
+  userId           Int?
+  eventTypeId      Int?
+  subscriberUrl    String
+  payloadTemplate  String?
+  createdAt        DateTime               @default(now())
+  active           Boolean                @default(true)
+  eventTriggers    WebhookTriggerEvents[]
+  user             User?                  @relation(fields: [userId], references: [id], onDelete: Cascade)
+  eventType        EventType?             @relation(fields: [eventTypeId], references: [id], onDelete: Cascade)
+  subscriptionType SubscriptionType       @default(WEBHOOK)
```

Comment:
Does it makes a difference from normal Webhooks? I'm struggling to understand the need to differentiate other than the payloadTemplate maybe?

Reply:
Gotcha! I have some improvements to make then. Since since the latest release we can just do `prisma.app.findUnique({ where: { slug: "zapier" }})` to check if Zapier is installed now. (We don't check `.env` variables anymore for that)